context/app/static/js/components/savedLists/SavedListPanel/hooks.ts

```diff
@@ -0,0 +1,40 @@
+import { useMemo } from 'react';
+import { SavedEntity } from 'js/components/savedLists/types';
+import { getIDsQuery } from 'js/helpers/queries';
+import useSearchData from 'js/hooks/useSearchData';
+
+interface EntityCounts {
+  entity_counts: {
+    buckets: { doc_count: number; key: string }[];
+  };
+}
+function useSavedEntityCountsData(savedEntities: Record<string, SavedEntity>) {
+  const query = useMemo(
+    () => ({
+      size: 0,
+      query: getIDsQuery(Object.keys(savedEntities)),
+      aggs: {
+        entity_counts: {
+          terms: { field: 'entity_type.keyword', size: 10 },
```

Comment:
```suggestion
          terms: { field: 'entity_type.keyword', size: 10000 },
```
I would just use the max value to be safe. If for whatever reason the number of `entity_type` variations were to exceed 10 the aggregations might not include donor, sample or dataset.